docs/source/robots.txt

```diff
@@ -0,0 +1,6 @@
+User-agent: *
+Allow: /*/latest/
+Allow: /en/latest/   # Fallback for bots that don't understand wildcards
+Allow: /*/stable/
+Allow: /en/stable/   # Fallback for bots that don't understand wildcards
```

Comment:
I think you copied this from the astro project, but are we sure this syntax is correct? The url for AiiDA's documentation on RTD starts with `/projects/aiida-core/en/stable`. Should these rules include the `/projects/aiida-core/`?

Reply:
You are right! 
yes, assuming RTD will place it to the right place: `https://aiida.readthedocs.io/robots.txt` 
The paths inside this file are relative to the site domain, so:
```suggestion
Allow: /projects/aiida-core/en/latest/   
Allow: /projects/aiida-core/en/stable/   
```

In any case, from the build, I now realize RTD places this file in a wrong place:
`https://aiida.readthedocs.io/projects/aiida-core/robots.txt` 

This makes it discoverable by search engines...

Reply:
Alright! 
One solution is to set an exact redirect in RTD web setting tab, 
from  `/robots.txt` to `/projects/aiida-core/robots.txt`

@sphuber I don't have access to RTD settings, may I ask you for this? if you agree.

Reply:
> @sphuber I don't have access to RTD settings, may I ask you for this? if you agree.

I have gone through the admin panel, but cannot find a way to customize the path. It says [in the docs](https://docs.readthedocs.io/en/stable/reference/robots.html) that ReadTheDocs automatically generates and serves. We can set older versions to hidden so that their paths are automatically included in the robots.txt to be excluded from indexing. Would that be a better approach as we are sure that the robots.txt is put in the correct place?

Reply:
Actually, hiding versions might not be ideal because some users may actually want to look at older version for some reason. We just don't want them to be indexed. And never mind, [the docs](https://docs.readthedocs.io/en/stable/reference/robots.html#tool-integration) also say what should be done for projects using Sphinx as we are, and it seems your approach is correct.